h2o-docs/src/booklets/v2_2015/source/sw/sections/usecase.tex

```diff
@@ -26,7 +26,7 @@ \subsection{Predicting Arrival Delay in Minutes - Regression}
 
     // Create SparkContext to execute application on Spark cluster
     val sc = new SparkContext(conf)
```

Comment:
`val sc = new SparkContext(conf)` should be replaced in this case by 
val spark = SparkSession.builder().config(conf).appName("SparkSessionZipsExample").getOrCreate()
```